### PR TITLE
15th july

### PR DESCRIPTION
**Pull request by springcoil**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,51 @@
...
+### OpenAPI
+* We launched [OpenAPI](https://aflorithmic.github.io/openapi/) so you can see in [Swagger](https://swagger.io/) style our endpoints. If you're used to this workflow let us know we'd love to talk some more about it. Feel free to reach out to peadar[at]aflorithmic[dot]ai
+
+### SuperOrg
+A common customer request was - I want to understand what my customers usage is. For example if you're a distribution partner or agency you'll encounter this workflow. 
+
+This allows you to see usage of your child organisations :) Have a look below. It's in production so you can 
+![SuperOrg](images/superorg_with_childorgs_usage.png) 
+### SyncTTS
+* When you use syncTTS with `metadata=True` there will also be a url generated
```
Comment (GetOnMyLvl97) [inline CHANGELOG.md:19]: Perfect! Does ETH knows about it already?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,51 @@
...
+
+```python 
+response = apiaudio.SyncTTS.create(text="Hello", voice="eth-einstein-v2", metadata=True)
```
Comment (GetOnMyLvl97) [inline CHANGELOG.md:22]: May we shouldn't use private customer voices in the changelog :D

**Review by GetOnMyLvl97 — CHANGES_REQUESTED** [review]
fix merge conflict pls:)

File: CHANGELOG.md (hunk 2 above)
Comment (springcoil) [reply CHANGELOG.md:22]: Ohh yeah what works as a voice?

[the remark this answers is not in the document]
Comment (springcoil) [reply CHANGELOG.md]: I'll change to shelly

**Review by GetOnMyLvl97 — APPROVED** [review]
LGTM

**Review by LindaKivine — APPROVED** [review]

**Pull request merged**